### ISSUE TITLE
Feature request: "All + Target's Party spread" targeting option

### ISSUE THREAD

**Issue by CronosZX** [ohrrpgce/ohrrpgce#1209]
Ran into this while while checking spread store targets today for an optional spread healing attack:
Optional spread+All (not dead) targets every hero and enemy in the field once spread is selected (makes sense since it's the target all setting).
But this means that there's currently no way of targeting both hero--[optional spread into]-->hero party and enemy---->enemy party with one target setting as of now (unless I'm unaware of it).

**Comment by rversteegen** [comment]
Right, optional spread is all or nothing. This is an interesting suggestion. I think it would be very rarely used though. The only use I can think of is a spread cure attack that can also be used to harm undead.

**Comment by bob-the-hamster** [comment]
I do remember wanting this many years ago, but just never getting around to figuring out the best way to do it.

Thinking about it now, I would probably implement it as a new target type "Optional Spread, Only One Type" which would only really be useful with the "All" target class

Optional Spread works by toggling "all" when you try to move the cursor off the edge, (or when you drag the mouse I think?) and "Optional Spread, Only One Type: could work the same way, just using the type of the single target previously selected, ally or enemy
